vm/chef/cookbooks/redash/files/redash-setup

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /opt/c2d/c2d-utils || exit 1
+
+readonly redash_base_path="/opt/redash"
+if [[ -e $redash_base_path/env ]]; then
+    rm $redash_base_path/env
+    touch $redash_base_path/env
+fi
+
+readonly redash_admin_name="$(get_attribute_value "redash_admin_name")"
+readonly redash_admin_password="$(get_attribute_value "redash_admin_password")"
+readonly redash_admin_email="$(get_attribute_value "redash_admin_email")"
+
+readonly redash_cookie_key="$(get_attribute_value "redash_cookie_key")"
+readonly redash_secret_key="$(get_attribute_value "redash_secret_key")"
+readonly redash_db_password="$(get_attribute_value "redash_db_password")"
+readonly redash_database_url="postgresql://postgres:${redash_db_password}@postgres/postgres"
+
+echo "PYTHONUNBUFFERED=0" >> $redash_base_path/env
+echo "REDASH_LOG_LEVEL=INFO" >> $redash_base_path/env
+echo "REDASH_REDIS_URL=redis://redis:6379/0" >> $redash_base_path/env
+echo "POSTGRES_PASSWORD=${redash_db_password}" >> $redash_base_path/env
+echo "REDASH_COOKIE_SECRET=${redash_cookie_key}" >> $redash_base_path/env
+echo "REDASH_SECRET_KEY=${redash_secret_key}" >> $redash_base_path/env
+echo "REDASH_DATABASE_URL=${redash_database_url}" >> $redash_base_path/env
+
+# Add compose variables to profile
+echo "export COMPOSE_PROJECT_NAME=redash" >> /etc/profile
+echo "export COMPOSE_FILE=/opt/redash/setup/docker-compose.yml" >> /etc/profile
+
+export COMPOSE_PROJECT_NAME=redash
+export COMPOSE_FILE=/opt/redash/setup/docker-compose.yml
+docker-compose run --rm server create_db
+docker-compose up -d
+
+# Wait for redash service and create admin user
+sleep 20
+curl -s -L localhost/setup -d "name=${redash_admin_name}&email=${redash_admin_email}&password=${redash_admin_password}&security_notifications=y&newsletter=y&org_name=default" > /dev/null
```

Comment:
`-d` expects encoded data
as you're passing raw emails and password, I guess it's safer to use `--data-urlencode`

also I'd break the line for readability